src/eon.erl

```diff
@@ -329,7 +329,7 @@ new_test() ->
   _           = new([{foo, 42}, {bar, 666}]),
   _           = new([{foo, bar},baz]),
   _           = new([foo,42, bar,666]),
-  {error, _}  = (catch new([foo,bar, baz])),
+  {'EXIT', _} = (catch new([foo,bar, baz])),
```

Comment:
not sure if that's what it's supposed to return, but that's what it returns anyway 😄